pkg/rook/logs.go

```diff
@@ -0,0 +1,19 @@
+package rook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+func SetCephLogLevel(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, storageClusterNamespace string, component, subsystem, level string) {
+
+	cephArgs := []string{"config", "set", component, fmt.Sprintf("debug_%s", subsystem), level}
+
+	exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", cephArgs, operatorNamespace, storageClusterNamespace, true, false)
```

Comment:
I believe the last `false` param is for `exitOnError`. I believe we want that to be true? Or else we need to handle the return error to skip printing success.

Reply:
yes. Missed that. Making `returnoutput` to `false` and `exitOnError` to `true`.  

``` 
❯ bin/odf set ceph-log-level osd ms 6
Info: successfully set the log levels for "osd" subsystem "ms" as "6"

❯ bin/odf set ceph-log-level osd ms y
Error EINVAL: error parsing value: value must take the form N or N/M, where N and M are integers
Error: command terminated with exit code 22

❯ bin/odf set ceph-log-level osd fgs 6
Error EINVAL: unrecognized config option 'debug_fgs'
Error: command terminated with exit code 22

❯ bin/odf set ceph-log-level osds test 6
Error EINVAL: unrecognized config target 'osds'
Error: command terminated with exit code 22
```